src/pages/Dashboard/Dashboard.component.tsx

```
@@ -46,6 +47,7 @@ export const Dashboard = () => {
             >
                 <TransactionsTable />
             </SectionContainer>
+            <Footer></Footer>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)